src/layouts/Roadmap.tsx

```diff
@@ -103,7 +103,9 @@ export const roadmapComponents = {
   // MeetupList,
 }
 
-interface IProps extends PageContent, ChildOnlyProp {}
+interface IProps extends MdPageContent, ChildOnlyProp {
+  frontmatter: RoadmapFrontmatter
+}
 export const RoadmapLayout: React.FC<IProps> = ({ children, frontmatter, slug, tocItems }) => {
```

Comment:
`MdPageContent` contains a `frontmatter` property typed as `Frontmatter` which is a union type of any possible frontmatter schema... Inside the layout here we can then cast it as the layout-specific frontmatter schema needed to enforce typing

(pattern repeated on all layouts)